rest_framework/serializers.py

```diff
@@ -106,17 +105,20 @@ def __getstate__(self):
         return dict(self)
 
 
-class SortedDictWithMetadata(SortedDict):
+class OrderedDictWithMetadata(OrderedDict):
```

Comment:
we may as well leave the name of our class the same.
smaller change footprint, less chance of breaking existing projects.